dist/api_data.py

```diff
@@ -1218,10 +1226,17 @@ def __ge__(self, other):
         permit sharing between processes (will automatically start an RPC server for primary
         processes and use RPC for secondary processes). <b>Not yet supported in WiredTiger</b>''',
         type='boolean'),
-    Config('prefetch', 'false', r'''
+    Config('prefetch', '', r'''
         Enable automatic detection of scans by applications, and attempt to pre-fetch future
         content into the cache''',
-        type='boolean'),
+        type='category', subconfig=[
+        Config('available', 'false', r'''
```

Comment:
Apologies if this has already been discussed. I don't like the combo of `available` and `enabled` - I think `enabled` would make more sense if it was named something like `default`. In fact, the parsing code in `conn_api.c` already has a better name for it (`auto_on`).